### DISCUSSION TITLE
Linkare Plancia Energetica Istanza Remota HA

### DISCUSSION THREAD

**Discussion by Borghese** [indomus/forum#2816] [GENERALE - Discussioni generiche]
ciao a tutti,
ho finalmente terminato l'installazione e configurazione di HA nella seconda casa su un raspy pi4, che adesso piano piano domotizzerò (Al momento ho messo solo uno shelly EM per monitorare i consumi)
A casa mia (la principale) ho HA che gira su un mini pc NUC, e grazie all'add on My HA Remote ho visibilità di tutte le entità del nodo remoto della seconda casa
Sto quindi iniziando a costruirmi la dashboard con le entità dell'istanza remota da monitorare (così riesco a vedere tutto da un unica app senza dovermi spostare ogni volta) ma ho un dubbio : 

C'è modo di linkare la plancia energetica dell'istanza remota, su quella principale ? ho frugato in lungo e in largo ma non trovo come farlo...

**Comment by 50494554524F** [comment]
se si tratta solo di tenere le cose insieme creati una pagina html con frames e fai quello che vuoi in semplicità, oppure puoi usare i frame di ha se non sono troppo limitanti

**Reply by Borghese** [reply]
grazie @50494554524F ma intendi una pagina html nella dashboard di home assistant ?
Onestamente non ci avevo pensato... ora mi studio un pò, non mi è neanche chiarissimo che risultato si possa ottenere...

**Reply by 50494554524F** [reply]
si, webpage di ha, ops avevo riscritto frame... oppure proprio una pagina html dove ci metti tutte le pagine ha che vuoi ed è meno limitante di una webpage

**Reply by Borghese** [reply]
ok provo a giocarci...  ti faccio sapere intanto grazie!
